content/blog/2020-04-16-april-20-community-gems.md

```diff
@@ -0,0 +1,108 @@
+---
+title: April '20 Community Gems
+date: 2020-04-16
+description: |
+  Great discussions and technical Q&A's from our users.
+descriptionLong: |
+  Look here every month for great discussions and technical Q&A's from our users 
+  and core development team.
+picture: ../../static/uploads/images/2020-04-16/DVC_Gems_April_20.png
+author: ../authors/elle_obrien.md
+commentsUrl: https://discuss.dvc.org/t/march-20-community-gems/336
+tags:
+  - Discord
```

Comment:
only one tag this time?

Reply:
forgot to fill in! Have added:
```
  - cache
  - concurrency
  - cloud
  - GitHub
```

Reply:
Let's not put very specific tags from some questions/answers unless they are very important.

Let's keep it similar to other Gems?

also, we should probably titelize all names?

Reply:
sure, have made it less specific and title-ized: 

`tags:
  - Discord
  - Gems
  - Pipelines`